source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+    |                                                | ``block:cyclic:cyclic``, see ``man srun``  |
+    |                                                | for more information.                      |
+    |                                                | Currently, the distribution setting for    |
+    |                                                | cores (the third "<value>" entry) has no   |
+    |                                                | effect on Frontier                         |
+    +------------------------------------------------+--------------------------------------------+
+    |  ``--ntasks-per-node=<ntasks>``                | If used without ``-n``: requests that a    |
+    |                                                | specific number of tasks be invoked on     |
+    |                                                | each node.                                 |
+    |                                                | If used with ``-n``: treated as a          |
+    |                                                | *maximum* count of tasks per node.         |
+    +------------------------------------------------+--------------------------------------------+
+
+********
+Software
+********
+
+Gaea has several software and libraries available.  These are typically
+available using the :ref:`Lmod module system <gaea-environment-modules>` to
+make the software available.  Use the `module avail` and `module spider`
+commands to see the available software.  Only modules in the :file:`/opt` and
+:file:`/sw` areas are supported at the RDHPCS level.  Projects and users can
+install software and software stacks in their user or project spaces, that is
+in :file:`/ncrc/home[12]/$USER/`, :file:`/usw`, and :file:`/ncrc/proj`
+locations, is maintained and supported by the users and projects that have made
+them available.
+
+*********
+Debugging
+*********
+
+Linaro DDT
+==========
+
+Linaro DDT is an advanced debugging tool used for scalar, multi-threaded, and
+large-scale parallel applications. In addition to traditional debugging
+features (setting breakpoints, stepping through code, examining variables), DDT
+also supports attaching to already-running processes and memory debugging.
+In-depth details of DDT can be found in the official `DDT User Guide
+<https://docs.linaroforge.com/latest/html/forge/forge/introduction_to_forge/ddt.html>`_,
+and instructions for how to use it on RDHPCS systems can be found on the
+:doc:`Debugging Software </software/debuggers/index>` page. DDT is the RDHPCS's
+recommended debugging software for large parallel applications.
+
+One of the most useful features of DDT is its remote debugging feature. This
+allows you to connect to a debugging session on RDHPCS systems from a client
+running on your workstation. The local client provides much faster interaction
+than you would have if using the graphical client on RDHPCS systems. For
```

Comment:
recommend:'

If you use, rather than "if using"

Reply:
fixed in a new push.